enaml/src/colorext.cpp

```diff
@@ -163,14 +163,14 @@ Color_getset[] = {
 static PyObject*
 Color__reduce__( Color* self, void* context )
 {
-    PyObject* cls = PyObject_Type(reinterpret_cast<PyObject*>( self ));
-    if ( !cls )
+    PyObject* obj = reinterpret_cast<PyObject*>( self );
```

Comment:
This cast will never fail. No need to check for `if( !obj )`.

Reply:
Thanks amended the commit to remove it.